cmd/statusd/library.go

```diff
@@ -323,10 +323,17 @@ func InitJail(js *C.char) {
 	statusAPI.JailBaseJS(C.GoString(js))
 }
 
-//Parse creates a new jail cell context and executes provided JavaScript code
+//Parse is DEPRECATED. Use CreateAndInitCell instead.
 //export Parse
 func Parse(chatID *C.char, js *C.char) *C.char {
-	res := statusAPI.JailParse(C.GoString(chatID), C.GoString(js))
+	res := statusAPI.CreateAndInitCell(C.GoString(chatID), C.GoString(js))
+	return C.CString(res)
+}
+
+//CreateAndInitCell creates a new jail cell context and executes provided JavaScript code.
+//export CreateAndInitCell
+func CreateAndInitCell(chatID *C.char, js *C.char) *C.char {
```

Comment:
1. Yeah, I found a few e2e tests that still use `Parse`,
2. Does it require any? If so, we need to provide one but I got an impression that it's not necessary. Unless `Parse` was used to call arbitrary JS,
3. Correct.

Reply:
1. There're still 4 such places, for example, this one: https://github.com/status-im/status-go/blob/e14b2331ebc4c26efd30e3df471a00d71586bbfc/cmd/statusd/utils.go#L1307
2. Yeah, I think so. Could you ask them about how they use `Parse` and if they need one?
3. It depends on the previous question but it may not be the case.

Reply:
1. Ugh, these cmd e2e tests... I will fix that.
2. I asked about it. We need to provide two bindings in such a case: `CreateAndInitCell` and `RunJS`. The first one will throw an error if a cell already exists. `RunJS` will just run raw JS code.
3. Answered in (2).

Reply:
Ok, so let's proceed with the second right away. At the end of this PR we'll send a list of incompatible changes.